README.md

```diff
@@ -225,11 +225,11 @@ First, decide if you are planning to build the sources for the 1.0 API or the 2.
 
 Projects built with the meson build-system require a separate build directory from the source path. The `meson` command will generate this directory for you.
 ```
-meson --buildtype=debug -Db_coverage=true -Dbuild_api_v1=false -Dbuild_api_v2=true  api2
+meson --buildtype=debug -Db_coverage=true  debugbuild
 ```
-The above command (run from the root of the source checkout) will create a new subdirectory - `api2` - configured to compile with debug symbols and `gcov` symbols to measure test coverage. It enables building the 2.0 API and skips the deprecated 1.0 API.
+The above command (run from the root of the source checkout) will create a new subdirectory - `debugbuild` - configured to compile with debug symbols and `gcov` symbols to measure test coverage. It enables building the 2.0 API and skips the deprecated 1.0 API.
```

Comment:
The "It enables building the 2.0 API and skips the deprecated 1.0 API." part is now extraneous and should probably be dropped as well.